README.md

```diff
@@ -21,15 +36,11 @@ positional arguments:
 * `environment` - An object containing your env vars (eg. `process.env`)
 * `validators` - An object that specifies the format of required vars.
 * `options` - An (optional) object, which supports the following keys:
-    * `strict` - (default: `false`) Enable more rigorous behavior. See "Strict Mode" below
-    * `reporter` - Pass in a function to override the default error handling and
-                   console output. See `src/reporter.js` for the default implementation.
-    * `transformer` - A function used to transform the cleaned environment object
-                      before it is returned from `cleanEnv`
-    * `dotEnvPath` - (default: `'.env'`) Path to the file that is parsed by
-                     [dotenv](https://github.com/motdotla/dotenv) to
-                     optionally load more env vars at runtime. Pass `null` if you want
-                     to skip `dotenv` processing entirely and only load from `process.env`.
+  * `reporter` - Pass in a function to override the default error handling and
+                 console output. See `src/reporter.js` for the default implementation.
+  * `middleware` - (optional) An array of functions that can modify the env object after it's
+                   validated and cleaned. Envalid ships with default middleware, but you can
+                   customize its behavior by supplying your own
 
 By default, `cleanEnv()` will log an error message and exit if any required
```

Comment:
```suggestion
By default, `cleanEnv()` will log an error message and throw an error if any required	
```

Reply:
Added some conditional reporter logic to use `process.exit(1)` in node and throw in other environments, with a README change to match. Let me know if you can see any issues with it